src/commands/login.ts

```diff
@@ -4,6 +4,10 @@ import Command from '../base-command';
 import inquirer = require('inquirer');
 
 export default class Login extends Command {
+  auth_required() {
+    return false;
```

Comment:
The init cmd should also be exempt. Are we missing any other commands?

Reply:
Should it though? We need to prefix the service name with an account name

Reply:
I think it handles that gracefully atm. Pretty sure David wanted it for non-auth.